source/community/groups/framework.md

```diff
@@ -77,24 +77,22 @@ Technical Specification Groups are the main working vehicle for adding new APIs
-  * Once the draft charter is acceptable to the participants in the discussion, a Call for Participation (CfP) is issued to appropriate lists (including [IIIF-Discuss][iiif-discuss] at the least) that announces the document and seeks the engagement of additional participants.
-    * Individuals from organizations across the IIIF community must respond to the CfP indicating that they are willing to take part and commit resources towards the Technical Specification Group's goals.
-    * At least three IIIF participating institutions  must respond positively, and no more than three institutions may respond negatively, for the group to be officially approved by the wider community.
-    * If fewer than three institutions are willing to contribute, then the group's topic is likely too specific and the work should be done outside of the Technical Specification Group process.  If more than three institutions object to the work being done, then there is a significant issue that should be resolved before committing resources.
-    * For Technical Specification Groups where deliverables would involve additions or changes to existing technical specifications, at least two current members of the editorial board must sign up as members of the group to ensure consistency with existing and ongoing work.
-  * At least two calendar weeks must pass between the CfP and official approval of the group.  If only one or two institutions are interested after four calendar weeks, then the CfP is considered closed and the original proposers of the charter should lead continued discussion and modify the charter before re-announcing.
-  * Once officially approved, the Community and Communications Officer will schedule, announce, and facilitate an initial planning call for the group. During the first call, the group will determine willingness and availability of volunteers to chair the group, agree upon group chairs, and determine a regular call schedule for moving forward.
-  * Technical Specification Groups should be documented on the iiif.io website, noting the following information:
+    6. The timeframe in which the specifications and implementations are needed and should be possible to come to consensus on
+  * If fewer than three institutions are willing to participate, then the group's topic is likely too specific and the work should be done outside of the Technical Specification Group process.  If more than three institutions object to the work being done, then there is a significant issue that should be resolved before moving forwards.
```

Comment:
Point to extension process?

Reply:
I don't think we need to cross reference similar processes. That could get unwieldy quickly. On the other hand, we should assess how consistent our processes are and whether they can be rationalized.

Reply:
Plus 1 to rationalizing the process in future once the extension process is documented.